.devcontainer/Dockerfile

```diff
@@ -1,7 +1,6 @@
-ARG VARIANT="3"
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT}
+ARG VARIANT=3
+FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
```

Comment:
Just for documentation:

See https://github.com/microsoft/vscode-dev-containers/blob/main/containers/python-3/README.md#configuration

Reply:
This changes also is from [official documentation](https://github.com/microsoft/vscode-dev-containers/blob/main/containers/python-3/.devcontainer/Dockerfile#L2C1-L2C1) but we put the latest version of Python.